src/core/runner.vala

```diff
@@ -0,0 +1,13 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private interface Games.Runner : Object {
+	public abstract Gtk.Widget? get_display ();
```

Comment:
Don't really like having a domain class depend on UI.
And aren't emulated games the only one that we are able to run in a Widget right now?
Does it even make sense for other (Desktop/Wine/Steam/Web) games?